src/commands/edge_app.rs

```diff
@@ -286,15 +288,20 @@ impl EdgeAppCommand {
         Ok(())
     }
 
-    pub fn run(&self, path: &Path, secrets: Vec<(String, String)>) -> Result<(), anyhow::Error> {
+    pub fn run(
+        &self,
+        path: &Path,
+        secrets: Vec<(String, String)>,
+    ) -> Result<(), anyhow::Error> {
         let address_shared = Arc::new(Mutex::new(None));
         let address_clone = address_shared.clone();
 
         let runtime = tokio::runtime::Runtime::new().unwrap();
         let path = path.to_path_buf();
         runtime.block_on(async {
             tokio::spawn(async move {
-                let address = run_server(path.as_path(), secrets).await.unwrap();
+                let secrets_val = secrets.iter().map(|(k, v)| { (k.clone(), EdgeAppServerValue::Str(v.clone()))}).collect();
```

Comment:
Shouldnt fmt format this line?

Reply:
Yeah forgot to run rustfmt while on my way out. Done.